GitUI/GitUICommands.cs

```diff
@@ -1391,206 +1387,168 @@ public void StartCreatePullRequest(IWin32Window owner, IRepositoryHostPlugin git
-            switch (args[1])
+            var command = args[1];
+            switch (command)
             {
                 case "about":
                     Application.Run(new FormAbout
                     {
                         StartPosition = FormStartPosition.CenterScreen
                     });
-                    return;
+                    return true;
                 case "add":
                 case "addfiles":
-                    StartAddFilesDialog(null, args.Count == 3 ? args[2] : ".");
-                    return;
+                    return StartAddFilesDialog(null, args.Count == 3 ? args[2] : ".");
                 case "apply":       // [filename]
                 case "applypatch":
-                    StartApplyPatchDialog(null, args.Count == 3 ? args[2] : "");
-                    return;
+                    return StartApplyPatchDialog(null, args.Count == 3 ? args[2] : "");
                 case "blame":       // filename
-                    RunBlameCommand(args);
-                    return;
+                    return RunBlameCommand(args);
                 case "branch":
-                    StartCreateBranchDialog();
-                    return;
+                    return StartCreateBranchDialog();
                 case "browse":      // [path] [-filter]
-                    RunBrowseCommand(args);
-                    return;
+                    return RunBrowseCommand(args);
                 case "checkout":
                 case "checkoutbranch":
-                    StartCheckoutBranch(null);
-                    return;
+                    return StartCheckoutBranch(null);
                 case "checkoutrevision":
-                    StartCheckoutRevisionDialog(null);
-                    return;
+                    return StartCheckoutRevisionDialog(null);
                 case "cherry":
-                    StartCherryPickDialog();
-                    return;
+                    return StartCherryPickDialog();
                 case "cleanup":
-                    StartCleanupRepositoryDialog();
-                    return;
+                    return StartCleanupRepositoryDialog();
                 case "clone":       // [path]
-                    RunCloneCommand(args);
-                    return;
+                    return RunCloneCommand(args);
                 case "commit":      // [--quiet]
-                    Commit(arguments);
-                    return;
-                case "difftool":      // filename
-                    Module.OpenWithDifftool(args[2]);
-                    return;
-                case "filehistory": // filename
+                    return Commit(arguments);
+                case "difftool":    // filename
+                    return Module.OpenWithDifftool(args[2]) == "";
+                case BlameHistoryCommand:
+                case FileHistoryCommand:
+                                    // filename [revision [--filter-by-revision]]
```

Comment:
Bad indentation

Reply:
It's not perfect, but it is aligned with the comments showing the argument lists of the other commands.